charts/hivemq-edge/tests/default/hivemq_edge_secret-license_test.yaml

```
@@ -0,0 +1,16 @@
+suite: HiveMQ Edge - Secret License tests
+
+templates:
+  - secret-license.yml
+release:
+  name: edge
+  namespace: hivemq-edge-namespace
+chart:
+  version: 0.1.1
+  appVersion: "2025.1"
+
+tests:
+  - it: Secret should not be present
+    asserts:
+      - isNull:
+          path: ""
```

This should check a bit more precisely that no K8s resource was created, and this is not a result of a failure.
```suggestion
      - notFailedTemplate: {}
      - hasDocuments:
          count: 0
```
Same can be applied for the same `isNull` checks in the other tests.

---

Done.